src/main/java/org/nuxeo/ecm/platform/sessioninspector/jsf/model/MonitorNode.java

```diff
@@ -192,11 +192,9 @@ public String getView() {
         return result;
     }
 
+    @Override
     public String toString() {
-        if (stateReference != null) {
-            return stateReference.toString();
-        }
-        return null;
+        return String.valueOf(stateReference);
```

Comment:
That's a slight change of behaviour, String.valueOf(null) returns a String `"null"` rather than a `null` value.
I assume that's fine.

Reply:
Yes that's the actual point of this change: `null` is not a valid return value for the `toString()` method.